include/deal.II/grid/grid_tools.h

```diff
@@ -983,6 +987,65 @@ namespace GridTools
   get_patch_around_cell(const typename Container::active_cell_iterator &cell);
 
 
+
+  /**
+   * This function returns a list of all parent cells of the given vector of active cells.
+   * To get the list of cells with the coarsest common level, the function first computes the minimum refinement level of all cells (min_level)
+   * then loops over all cells of the given input vector (patch_cells), for each cell of the loop
+   * if its level==min_level then the cell is pushed back to the vector of output cells,
+   * else the function looks for the parent cell of the given cell, with the level equal to the min_level,
+   * and pushes back this parent cell to the vector of out put cells.
+   *
+   * @tparam Container
+   * In C++, the compiler can not determine the type of <code>Container</code>
+   * from the function call. You need to specify it as an explicit template
+   * argument following the function name.
+   * @param[in] patch_cells A vector of active cells.
+   * @return A list of cells with the coarsest common level of refinement
+   *
+   * @author Arezou Ghesmati, Wolfgang Bangerth, 2015
+   */
+  template<class Container >
+  std::vector<typename Container::cell_iterator> get_cells_at_coarsest_common_level(
+    const std::vector<typename Container::active_cell_iterator> &patch_cells);
```

Comment:
They are active cells because that's what comes out of the function (added in an earlier commit months ago) that computes the "patch" around a cell -- they are active cells, and you get a vector of them, so that's the natural argument to the current function.